src/HTMLDocument.php

```diff
@@ -99,4 +104,14 @@ private function getOrCreateElement(string $tagName):Element {
 	return $element;
 }
 
+private function fillEmptyDocumentElement() {
```

Comment:
Might "scaffold" or "initialise" be a better verb for this method rather than "fill"?  
Rather than filling the DOMDoc this method is putting the bare minimum in there for it to be a valid DOM.